connect/runtime/src/main/java/org/apache/kafka/connect/runtime/errors/RetryWithToleranceOperator.java

```diff
@@ -331,6 +351,15 @@ public synchronized Throwable error() {
         return this.context.error();
     }
 
+    /**
+     * Exit from any currently ongoing retry loop and mark the operation as failed.
```

Comment:
This makes it sound like we can interrupt the operation that we're retrying, but that's not the case. And because Java doesn't really provide a great way to force a thread to stop or return control to a call site, we probably can't ever really accomplish that. I do like the part about marking the operation as failed, though--nice attention to detail there 👍

What about "Attempt no further retries for any operations, and mark any ongoing operations that are blocked during backoff, or currently being invoked and do not complete successfully, as failed"?

Reply:
That's great! LGTM 👍